Sources/SwiftProtobuf/Google_Protobuf_Wrappers+Extensions.swift

```diff
@@ -30,33 +30,6 @@ protocol ProtobufWrapper {
 
   /// Creates a new instance of the wrapper with the given value.
   init(_ value: WrappedType.BaseType)
-
-  /// Implements the JSON serialization logic for the wrapper types.
-  ///
-  /// We cannot have the `ProtobufWrapper` extension below implement
-  /// `serializeJSON` because it is also implemented in extensions to other
-  /// protocols. In other words, the compiler cannot disambiguate between them
-  /// because both extension implementations have equal "weight". Instead, we
-  /// have to override `serializeJSON` in the extensions to the generated
-  /// concrete structs -- since the struct extension is more specific than the
-  /// protocol extensions, it takes priority. In order to share the
-  /// implementation, we have those extensions "hop" to this one.
-  func serializeWrapperJSON() throws -> String
-}
-
-extension ProtobufWrapper {
-  // NOTE: The `init(_ value: WrappedType.BaseType)` initializer repeated below
-  // should theoretically be able to go here and be declared public, but this
-  // causes linker errors in release builds (see issue #70). If this is indeed a
-  // bug and should be allowed, we should move the initializer back into this
-  // extension once it's fixed, to reduce a small amount of code duplication/
-  // bloat.
-
-  func serializeWrapperJSON() throws -> String {
-    var encoder = JSONEncoder()
-    try WrappedType.serializeJSONValue(encoder: &encoder, value: value)
```

Comment:
I've not yet completely eliminated `serializeJSONValue`, but this is chipping away at it.